build.gradle.kts

```diff
@@ -48,3 +49,25 @@ for (projectName in listOf("tutorial-app", "helloworld-app", "cluster-demo", "me
         project(projectName).plugins.apply("designcompose.conventions.roborazzi")
     }
 }
+
+tasks.register<JacocoReport>("jacocoTestReport") {
+    dependsOn(subprojects.map { it.tasks.withType<Test>() })
+
+    val coreModules = listOf("annotation", "codegen", "common", "designcompose")
+    val source = files(subprojects.filter { it.name in coreModules }.map {
+        val sourceSets = it.extensions.findByType(SourceSetContainer::class.java)
+        sourceSets?.findByName("main")?.allSource
+    })
+    sourceDirectories.setFrom(source)
+    classDirectories.setFrom(files(subprojects.filter { it.name in coreModules }.map {
+        val sourceSets = it.extensions.findByType(SourceSetContainer::class.java)
+        sourceSets?.findByName("main")?.output
+    }))
+    executionData.setFrom(fileTree(project.rootDir) {
+        include("**/build/jacoco/test.exec")
```

Comment:
This might be too broad, but probably not a big deal.

Double check that the Jacoco plugin doesn't already have an "aggregate results" task or something like that